From d4554696dcc9cf218b98faecb1f521246cfe3f5e Mon Sep 17 00:00:00 2001
From: Richard Bradley <richard.bradley@joyent.com>
Date: Thu, 7 Dec 2017 10:46:33 +0000
Subject: [PATCH] MANATEE-341 "manatee-adm rebuild" hangs when rebuilding a
 sync peer MANATEE-344 Rebuild should not exit if postgres hasn't started on
 its first attempt MANATEE-359 manatee reports primary to sync replication as
 async Reviewed by: Dave Pacheco <dap@joyent.com> Approved by: Dave Pacheco
 <dap@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 11dbc67..ed8f2ea 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "manta": "1.2.6",
-        "manatee": "git+https://github.com/joyent/manatee.git#6a769d97",
+        "manatee": "git+https://github.com/joyent/manatee.git#50f8c783",
         "carrier": "0.2.0"
     },
     "sdcDependencies": {
-- 
2.21.0

